content/en/docs/Writing policies/jmespath.md

```diff
@@ -434,51 +442,1271 @@ spec:
+        - name: allpaths
+          apiCall:
+            urlPath: "/apis/networking.k8s.io/v1/ingresses"
+            jmesPath: "items[].spec.rules[].http.paths[].path"
+        # Looks up the Ingress paths in the same Namespace where the incoming request is targeted.
+        - name: nspath
+          apiCall:
+            urlPath: "/apis/networking.k8s.io/v1/namespaces/{{request.object.metadata.namespace}}/ingresses"
+            jmesPath: "items[].spec.rules[].http.paths[].path"
+      preconditions:
+        - key: "{{request.operation}}"
+          operator: Equals
+          value: "CREATE"
+      validate:
+        message: >-
+          The root path /{{request.object.spec.rules[].http.paths[].path | to_string(@) | split(@, '/') | [1]}}/ exists
+          in another Ingress rule elsewhere in the cluster.
+        deny:
+          conditions:
+            all:
+              # Deny if the root path of the request exists somewhere else in the cluster other than the same Namespace.
+              - key: /{{request.object.spec.rules[].http.paths[].path | to_string(@) | split(@, '/') | [1]}}/
+                operator: In
+                value: "{{allpaths}}"
+              - key: /{{request.object.spec.rules[].http.paths[].path | to_string(@) | split(@, '/') | [1]}}/
+                operator: NotIn
+                value: "{{nspath}}"
+```
+
+</p>
+</details>
+
+### Subtract
+
+<details><summary>Expand</summary>
+<p>
+
+The `subtract()` filter performs arithmetic subtraction capabilities between two input fields (terms) and produces an output difference. Like other arithmetic custom filters, it is input aware based on the type passed and, for quantities, allows auto conversion between units of measure. For example, subtracting 10Mi (ten mebibytes) minus 5Ki (five kibibytes) results in the value 10235Ki. The `subtract()` filter is currently under development to better account for all permutations of input types, however the below table captures the most common and practical use cases.
+
+Arithmetic filters like `subtract()` currently accept inputs in the following formats.
+
+* Number (ex., \`10\`)
+* Quantity (ex., '10Mi')
+* Duration (ex., '10h')
+
+Note that how the inputs are enclosed determines how Kyverno interprets their type. Numbers enclosed in back ticks are scalar values while quantities and durations are enclosed in single quotes thus treating them as strings. Using the correct enclosing character is important because, in Kubernetes "regular" numbers are treated implicitly as units of measure. The number written \`10\` is interpreted as an integer or "the number ten" whereas '10' is interpreted as a string or "ten bytes". See the [Formatting](#formatting) section above for more details.
+
+| Input 1            | Input 2            | Output   |
+|--------------------|--------------------|----------|
+| Number             | Number             | Number   |
+| Quantity           | Number             | Quantity |
```

Comment:
I don't think this mathematically makes sense and I wonder if we should even document this.

Reply:
Will remove for now.